lib/cli/parse.rs

```diff
@@ -771,6 +772,37 @@ pub(super) fn purse_identifier(purse_id: &str) -> Result<PurseIdentifier, CliErr
     Ok(PurseIdentifier::MainPurseUnderPublicKey(public_key))
 }
 
+/// Add
```

Comment:
This comment is strange. I am looking were tests are affected by this change on get_info as it seems to me get_info only receives old public key so is parse::account_identifier(account_identifier) well tested with an account hash ?

Reply:
I'll add unit tests here for this method. It's well tested manually but not in the code.